include/openssl/ssl.h

```diff
@@ -1688,7 +1688,10 @@ SSL_SESSION *d2i_SSL_SESSION(SSL_SESSION **a, const unsigned char **pp,
                              long length);
 
 # ifdef HEADER_X509_H
-__owur X509 *SSL_get_peer_certificate(const SSL *s);
+__owur X509 *SSL_get0_peer_certificate(const SSL *s);
+__owur X509 *SSL_get1_peer_certificate(const SSL *s);
+/* Deprecated in 3.0.0 */
```

Comment:
``` C
# if OPENSSL_API_LEVEL < 3
```

Reply:
I did something like this before and @mattcaswell asked me to change it...
https://github.com/openssl/openssl/pull/8686#discussion_r282399694

Reply:
So, which should it be? @mattcaswell @levitte 
```
# if !OPENSSL_API_3
```
or
```
# if OPENSSL_API_LEVEL < 3
```

Reply:
It should be `#ifndef OPENSSL_NO_DEPRECATED_3_0`

Reply:
This has been going back-and-forth for a while. "Final answer?"

Reply:
This is the final answer. To verify please look at how other things are deprecated - for example the various low-level crypto functions for example in include/openssl/aes.h.